src/main/java/org/opentripplanner/apis/gtfs/datafetchers/RentalVehicleImpl.java

```diff
@@ -61,6 +62,11 @@ public DataFetcher<RentalVehicleType> vehicleType() {
     return environment -> getSource(environment).vehicleType;
   }
 
+  @Override
```

Comment:
We don't want to expose core model classes directly in the API. Can you please add an Impl (=mapper) class just like `RentalVehicleImpl`?

Reply:
I have added a mapper.